docker/hooks/build

```diff
@@ -6,4 +6,8 @@
 # ensure that "COPY . /src" is referring to the repo root, not the directory
 # that contains the Dockerfile.
 # This script gets executed with a pwd of wherever the Dockerfile is.
-docker build --build-arg BUILD_CONCURRENCY=${CONCURRENCY:-1} --build-arg DOCKER_TAG=${DOCKER_TAG} -t $IMAGE_NAME -f Dockerfile ..
+
+DOCKER_BUILD="docker build --build-arg BUILD_CONCURRENCY=${CONCURRENCY} --build-arg DOCKER_TAG=${DOCKER_TAG:?unset} -t ${IMAGE_NAME:?unset} -f"
+
+$DOCKER_BUILD Dockerfile ..
+$DOCKER_BUILD Dockerfile-alpine ..
```

Comment:
Actually this file is not used anymore (at least as far as I understand). 

During release we build images without using it, see: https://github.com/Project-OSRM/osrm-backend/blob/7ffc08be2849742ceb92fbdc756b7af3ba6f8f55/.github/workflows/osrm-backend-docker.yml#L56

Also we have this for CI: https://github.com/Project-OSRM/osrm-backend/blob/7ffc08be2849742ceb92fbdc756b7af3ba6f8f55/.github/workflows/osrm-backend.yml#L119


In general IMO your PR is already very welcome and I am ready to merge it, but if you have some time would be great to also update these CI jobs (may be at least [this one](https://github.com/Project-OSRM/osrm-backend/blob/7ffc08be2849742ceb92fbdc756b7af3ba6f8f55/.github/workflows/osrm-backend.yml#L119) to make sure that this Alpine image is buildable after each PR).

Reply:
It's handy to have something around to remind you how to build it locally.
Maybe thats the reason why it's still around?

👍 I'll look into the CI jobs. Only built it locally so far.

Reply:
It seems until you merge at least one PR you will have to wait for my approval before running CI, so it will make feedback loop for you quite long... We can merge Dockerfile only first (and then add CI in separate PR) if it will be a problem - just let me know.

Reply:
Fine with me. I just removed the last commit with the workflow changes.